packages/react-native-aztec/ios/RNTAztecView/RCTAztecView.swift

```diff
@@ -358,10 +364,14 @@ class RCTAztecView: Aztec.TextView {
     }
 
     public override func insertDictationResult(_ dictationResult: [UIDictationPhrase]) {
-        let objectPlaceholder = "\u{FFFC}"
         let dictationText = dictationResult.reduce("") { $0 + $1.text }
-        isInsertingDictationResult = false
-        self.text = self.text?.replacingOccurrences(of: objectPlaceholder, with: dictationText)
+        if #available(iOS 16, *) {
+            insertText(dictationText)
+        } else {
+            let objectPlaceholder = "\u{FFFC}"
+            isInsertingDictationResult = false
+            self.text = self.text?.replacingOccurrences(of: objectPlaceholder, with: dictationText)
```

Comment:
It's necessary to keep this in place for older version, as without it an `obj` icon is added along with dictated text.

![Simulator Screen Shot - iPhone 6s (15 5) - 2023-03-15 at 17 33 15](https://user-images.githubusercontent.com/2998162/225433984-96f00047-096a-448a-afa4-cbd734c6c99b.png)